src/http.rs

```diff
@@ -401,6 +401,10 @@ pub struct KeepaliveConfig {
     /// The maximum amount of time a connection may exist before it is closed
     /// by sending a `Connection: close` header on the HTTP response.
     ///
+    ///
+    /// Set this to a high number to disable this feature (in the future setting this to `null`
```

Comment:
```suggestion
    /// Set this to a large value to disable this feature (in the future setting this to `null`
```

Or `large number` ?

Also wondering if an example of the large number would be useful.

Reply:
Good call, `large` is better than `high` 😄 I'll add an example too.